Cargo.toml

```diff
@@ -35,13 +35,16 @@ path = "src/main.rs"
 cast = "0.2.2"
 clap = "2.26.0"
 env_logger = "~0.5"
-error-chain = "0.11.0"
 inflections = "1.1.0"
 log = { version = "~0.4", features = ["std"] }
 quote = "1.0"
 proc-macro2 = "1.0"
+anyhow = "1.0.19"
+thiserror = "1.0.5"
 
 [dependencies.svd-parser]
+git = "https://github.com/rust-embedded/svd"
+branch = "anyhow"
```

Comment:
Assuming this is just for the draft.

We could use the new git+version support added for publishing crates added recently, but I'm not sure myself how to properly use it.